specificatie/BRK-Bevragen/genereervariant/openapi.yaml

```diff
@@ -4267,12 +4264,13 @@ components:
             op deze kadastaraal onroerende zaak
           items:
             type: string
-      description: Een kadastraal onroerende zaak is een perceel of een appartementsrecht.
-        Een perceel is een stuk grond waarvan het Kadaster de grenzen heeft gemeten
-        en dat bij het Kadaster een eigen nummer heeft. Een appartementsrecht is het
-        recht dat iemand of een organisatie heeft om eigenaar te zijn van een deel
-        van een gebouw of een stuk grond. In de praktijk wordt dit deel meestal een
-        appartement genoemd.
+      description: "Een kadastraal onroerende zaak is een perceel of een appartementsrecht.\
```

Comment:
Ik zie dat er verschillende manieren van descriptions worden gebruikt:
- tekst tussen dubbele aanhalingstekens en \ tekens
- tekst tussen enkele aanhalingstekens
- |- na de description: attribuut en vervolgens tekst in de volgende regels

Is het zinvol om ook hier één vorm te kiezen t.b.v. consistentie?